packages/adapter-sveltekit/src/hooks/documentation-read.ts

```diff
@@ -32,8 +32,8 @@ export const documentationRead: DocumentationReadHook<PluginOptions> = async (
 			dataFileContent = source`
 				import { createClient } from "$lib/prismicio";
 
-				export async function load({ params, fetch }) {
-					const client = createClient({ fetch });
+				export async function load({ params, fetch, cookies }) {
```

Comment:
We need to add the cookies manually now?

Reply:
Yes, the client needs the request's cookies to check if a preview is active and to retrieve the preview ref. Content is still fetched on the server during a preview, which requires the preview ref.

Next.js projects don't need to explicitly pass cookies because Next.js exposes a request's cookies via the `cookies()` function, which reads data implicitly using `AsyncLocalStorage` (aka magic).